File: dsa/array_string_obj/isPrime.js
let n = 7;
let isPrime = true;
if (n === 1 || n < 1) {
  console.log("prime starts from 2");
} else {
  for (let i = 2; i < n; i++) {
    if (n % i === 0) {
      isPrime = false;
      break;

    }
  }
  if (isPrime) {
    console.log("prime"); //return true
  } else {
    console.log("not a prime");
  }
}

File: dsa/array_string_obj/isPrime.py
n = 7
if n <= 1:
    print("prime starts from 2")
else:
    is_prime = True
    for i in range(2, n):
        if n % i == 0:
            is_prime = False
            break
    if is_prime:
        print("prime")
    else:
        print("not a prime")
